frontend/src/metabase/query_builder/components/QueryModals/QueryModals.tsx

```diff
@@ -42,7 +39,6 @@ interface QueryModalsProps {
   question: Question;
   updateQuestion: (question: Question, config?: UpdateQuestionOpts) => void;
   setQueryBuilderMode: (mode: QueryBuilderMode) => void;
-  setUIControls: (opts: Partial<QueryBuilderUIControls>) => void;
```

Comment:
I assume this is no longer used in the main app?

Reply:
yep!